openadmet/models/anvil/workflow.py

```diff
@@ -139,6 +156,15 @@ class ModelSpec(AnvilSection):
     section_name: ClassVar[str] = "model"
 
 
+class EnsembleSpec(AnvilSection):
+    """
+    Ensemble specification.
+    """
+
+    section_name: ClassVar[str] = "ensemble"
+    optional: bool = True
```

Comment:
Shouldn't be a need for this, you can just use `Optional[type]` = None` if you want it off by default., e.g https://github.com/OpenADMET/openadmet-models/pull/272/files#diff-a054b503977ac8be105f04344622556d2f9d17e6ac5ff38654fe2412a296f62aR305